contracts/adapters/swap/swap-baby/src/swap_baby.rs

```diff
@@ -0,0 +1,62 @@
+use cosmwasm_schema::{cw_serde, QueryResponses};
+use cosmwasm_std::{to_json_binary, Coin, CosmosMsg, Decimal, StdResult, Uint128, WasmMsg};
+
+#[cw_serde]
+pub enum ExecuteMsg {
```

Comment:
Question to learn: Does swap baby plan to be open sourced / in a GH repo we can link to instead of copying over the types, or is this the best way forward (no plans to release the code)

Reply:
Soon to be open sourced!